test/config.test.ts

```diff
@@ -0,0 +1,77 @@
+describe("environmental variables", () => {
```

Comment:
Should we add a test case for the `env` environment variable (when it is explicitly set and when it is default)? coverage is showing not all branches are tested.

Reply:
We left out a test case for this because jest is setting the env for itself to "test"